TYPO3.Flow/Scripts/flow.php

```diff
@@ -23,6 +23,12 @@
     if (DIRECTORY_SEPARATOR !== '/') {
         exit('The core:setfilepermissions command is only available on UNIX platforms.' . PHP_EOL);
     }
+    
+    $filePermissions = decoct(fileperms(__DIR__ . '/setfilepermissions.sh') & 0777);
+    if ($filePermissions !== '700'){
```

Comment:
hey thanks for keeping updating this one, there one tiny nitpick left.. should be a space between ``)`` and ``{`` as in ``if ($filePermissions !== '700') {``.. you can just go to https://github.com/wexi79/flow-development-collection/edit/patch-4/TYPO3.Flow/Scripts/flow.php, fix it and submit an additional commit to the branch and it will automatically update this PR.. no need to close the pr and such